x-pack/plugins/cases/common/types/api/case/v1.test.ts

```diff
@@ -259,6 +290,26 @@ describe('Status', () => {
         PathReporter.report(CasePostRequestRt.decode({ ...defaultRequest, category }))
       ).toContain('The length of the category is too long. The maximum length is 50.');
     });
+
+    it('removes foo:bar attributes from customFields', () => {
+      const customField = defaultRequest.customFields ? defaultRequest.customFields[0] : {};
```

Comment:
nit: I don't think we need this check here as the custom fields are set in L174.

Reply:
typescript complains otherwise that it is possibly `undefined` 😬